fragment.go

```diff
@@ -1798,7 +1798,7 @@ func (f *fragment) importValue(columnIDs, values []uint64, bitDepth uint, clear
 func (f *fragment) importRoaring(data []byte, clear bool) error {
 	f.mu.Lock()
 	defer f.mu.Unlock()
-	bm := roaring.NewBitmap()
+	bm := roaring.NewBTreeBitmap()
```

Comment:
This seems reasonable, but I also wonder: Should the default be changed such that NewBitmap makes the kind we want in the first place?

Reply:
the kind we want, in theory, changes based on the usage. After briefly reviewing our usage, I think the safest thing to do would be to change the default... however, I'd like to look into it a bit more, run benchmarks, etc. Can we save it for another PR?

Reply:
yeah, that seems reasonable. i am not 100% sure that we want to actually have multiple kinds configured at once, but on reflection, it's a good idea to do that and test it that way, because that way we don't risk breaking it so that we couldn't later if we had a good reason to. (i'm thinking about some fancy stuff for distinguishing probably-read-only data from data that expects updates).